src/main/kotlin/icfp2019/app.kt

```diff
@@ -10,7 +10,7 @@ fun main(args: Array<String>) {
             println("Running " + it.name)
             val problem = parseDesc(it.readText())
             val solution = solve(problem)
-            File(it.nameWithoutExtension + ".sol").writeBytes(solution.toByteArray())
+            File("./problems/" + it.nameWithoutExtension + ".sol").writeBytes(solution.toByteArray())
```

Comment:
Use the Path class (represented by it) to get the full path, the name without extension and then add the extension.  Don't hard code the location.